src/pal/tools/clang-compiler-override-arm.txt

```diff
@@ -6,7 +6,7 @@ SET (CLR_C_FLAGS_CHECKED_INIT          "-g -O1")
 SET (CMAKE_C_FLAGS_RELEASE_INIT        "-g -O1")
 SET (CMAKE_C_FLAGS_RELWITHDEBINFO_INIT "-g -O1")
 
-SET (CMAKE_CXX_FLAGS_INIT                "-Wall -Wno-null-conversion -std=c++11")
```

Comment:
We want to be on path to enable more code sharing with Mono. Enabling C++17 is not helping with it. It would be actually desirable to go in opposite direction and depend on fewer recent C++ features to enable easier code sharing with Mono.

Mono is plain C today, and they are looking at whether they can adopt a subset of C++11. They are not going in position to adopt C++17 in foreseeable future.

Reply:
cc @marek-safar